environment.yml

```diff
@@ -32,3 +32,6 @@ dependencies:
   - license-expression
   - libarchive
   - cirun
+  - pydantic >=1.8.1,<2
```

Comment:
if there isn't a super-compelling reason, getting on the pydantic 2 bus probably makes sense.